cmake/modules/FindOpenACC.cmake

```diff
@@ -0,0 +1,104 @@
+#.rst:
```

Comment:
CMake 3.10 has added a FindOpenACC script so this should not be necessary anymore.

Reply:
From the PIConGPU side it's ok if Alpaka moves to CMake 3.10+ as well.

Reply:
With regards to #637, this new file will need an MPL-2.0 header :)